code/modules/mob/living/simple_animal/familiars/familiars.dm

```diff
@@ -189,10 +155,44 @@
 	holder_type = /obj/item/weapon/holder/cat
 	mob_size = MOB_SMALL
 
-	health = 25
-	maxHealth = 25
+	health = 45
+	maxHealth = 45
 	melee_damage_lower = 3
 	melee_damage_upper = 4
 	attacktext = "clawed"
+	density = 0
 
 	wizardy_spells = list(/spell/targeted/subjugation)
+
+
+/mob/living/simple_animal/mouse/familiar
```

Comment:
Retyped to inherit from mouse so it takes on all of the base functionality of said animal. Ie, behaves like a mouse with some extra stuff. It was easier than duplicating lots of mouse code.

Should probably do the same with other familiars, but most of their base animals lack much unique behaviour to be worth it. i corrected several mob sizes and densities on them though